src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,46 @@
+package qna.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.ForeignKey;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+
+@Embeddable
+public class Answers {
+
+	@OneToMany(cascade = CascadeType.ALL)
+	@JoinColumn(name = "question_id", foreignKey = @ForeignKey(name = "fk_answer_to_question"))
+	private List<Answer> answers = new ArrayList<>();
+
+	public Answers() {
+	}
+
+	public void add(Answer answer) {
+		this.answers.add(answer);
+	}
+
+	public boolean contains(Answer answer) {
+		return this.answers.contains(answer);
+	}
+
+	public void deleteAnswer(){
+		this.answers.stream()
+			.forEach(answer -> answer.setDeleted(true));
+	}
```

Comment:
다음과 같이 해볼 수 있을 것 같아요! 😄 
```suggestion
	public void deleteAnswer(){
		this.answers.forEach(answer -> answer.setDeleted(true));
	}
```